jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ProbeHandlerTest.java

```diff
@@ -243,17 +239,17 @@ public void getTCPWithHTTPURLAndWithoutPort() {
 
         probe = probeHandler.getProbe(probeConfig);
         //assertion
-        assertNotNull(probe);
-        assertNotNull(probe.getHttpGet());
-        assertNull(probe.getTcpSocket());
-        assertNull(probe.getExec());
-        assertEquals(5,probe.getInitialDelaySeconds().intValue());
-        assertEquals(5,probe.getTimeoutSeconds().intValue());
-        assertEquals("www.healthcheck.com",probe.getHttpGet().getHost());
-        assertNull(probe.getHttpGet().getHttpHeaders());
-        assertEquals("/healthz",probe.getHttpGet().getPath());
-        assertEquals(8080,probe.getHttpGet().getPort().getIntVal().intValue());
-        assertEquals("HTTP",probe.getHttpGet().getScheme());
+        assertThat(probe).isNotNull();
+        assertThat(probe.getHttpGet()).isNotNull();
+        assertThat(probe.getTcpSocket()).isNull();
+        assertThat(probe.getExec()).isNull();
+        assertThat(probe.getInitialDelaySeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getTimeoutSeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getHttpGet().getHost()).isEqualTo("www.healthcheck.com");
+        assertThat(probe.getHttpGet().getHttpHeaders()).isNull();
+        assertThat(probe.getHttpGet().getPath()).isEqualTo("/healthz");
+        assertThat(probe.getHttpGet().getPort().getIntVal().intValue()).isEqualTo(8080);
+        assertThat(probe.getHttpGet().getScheme()).isEqualTo("HTTP");
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.